massgen/tests/test_evolving_criteria.py

```
@@ -0,0 +1,831 @@
+    from massgen.evaluation_criteria_generator import parse_evolution_response
+
+    current = _make_criteria(3)
+    evolved_item = {
+        "id": "E1",
+        "text": "Elevated E1 text — must be deeply original",
+        "category": "primary",
+        "anti_patterns": ["generic text"],
+        "score_anchors": {"3": "boring", "5": "ok", "7": "interesting", "9": "outstanding"},
+    }
+    response = _make_evolution_json(unchanged_ids=["E2", "E3"], evolved=[evolved_item])
+    criteria, summary, is_unchanged = parse_evolution_response(response, current)
```

Ruff (0.15.9)

[warning] Unpacked variable `summary` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)